org.kde.klevernotes.json

```diff
@@ -0,0 +1,60 @@
+{
+    "id": "org.kde.klevernotes",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.7",
+    "sdk": "org.kde.Sdk",
+    "base": "io.qt.qtwebengine.BaseApp",
+    "base-version": "6.7",
+    "command": "klevernotes",
+    "finish-args": [
+        "--device=dri",
+        "--env=QML_IMPORT_PATH=/app/qml",
+        "--filesystem=home",
```

Comment:
Should be removed unless it clearly does not work without (and a bug should be filed then).

Reply:
You're right, thing seems to work fine without `"--filesystem=home"`. 
Only the displayed path to the storage seems to be affected but that's no big deal.